docs/tutorial/part-four/index.md

```diff
@@ -242,39 +243,44 @@ module.exports = {
 }
 ```
 
-Restart the development server.
+Restart the development server. Now, you can access the `title` with a GraphyQl
+query. A `StaticQuery` allows you to query from anywhere inside your source
+code. There are some other differences, you can [read more about them
+here](/docs/static-query/). To query for the `title` you'd use:
 
-Then edit the two components:
-
-`src/pages/about.js`
-
-```jsx{2,5,7,14-23}
-import React from "react"
-import { graphql } from "gatsby"
-import Layout from "../components/layout"
+```
+{
+  site {
+    siteMetadata {
+      title
+    }
+  }
+}
+```
 
-export default ({ data }) => (
-  <Layout>
-    <h1>About {data.site.siteMetadata.title}</h1>
-    <p>
-      We're the only site running on your computer dedicated to showing the best
-      photos and videos of pandas eating lots of food.
-    </p>
-  </Layout>
-)
+In your page, you would add a query inside a `<StaticQuery/>` tag somewhere in
+your source code, for example:
 
-export const query = graphql`
-  query {
-    site {
-      siteMetadata {
-        title
+```jsx
```

Comment:
I would remove this section, and leave the example usage just in the code snippet for the `layout.js` change.

Reply:
The reason I put that in is that the section is all about GraphQL but we don't actually show a simple query. Moreover, the simple query syntax is buried in the format of the page/StaticQuery.  When readers are moving fast, they often miss the fine points...For example, I only recall one mention somewhere that users can test queries out in `http://localhost:8000/___graphql` --- this seems like a good point to illustrate it again.

Reply:
I agree -- my comment was about the `... source code ...` Static Query snippet below. Id take that out and just show the usage in `layout.js` -- not include this extra section.

Reply:
Ah, gotcha, I think I've got it right.